src/main/java/games/strategy/engine/framework/ui/GameChooserModel.java

```diff
@@ -188,13 +188,16 @@ private static void addNewGameChooserEntry(final List<GameChooserEntry> entries,
     }
   }
 
-  public GameChooserEntry findByName(final String name) {
+  /**
+   * Searches for a GameChooserEntry whose gameName matches the input parameter.
+   */
+  public Optional<GameChooserEntry> findByName(final String name) {
     for (int i = 0; i < size(); i++) {
       if (get(i).getGameData().getGameName().equals(name)) {
-        return get(i);
+        return Optional.of(get(i));
       }
     }
-    return null;
+    return Optional.empty();
```

Comment:
Suggestion:
```java
return IntStream.range(0, size())
  .mapToObj(this::get)
  .filter(e -> e.getGameData.getGameName().equals(name))
  .findAny(); // Or findFirst if order does matter
```

Reply:
Good suggestion to use an int stream 👍   Done in: f65e0950a